src/page-modules/assistant/trip/index.tsx

```diff
@@ -0,0 +1,177 @@
+import { PageText, useTranslation } from '@atb/translations';
+import { Typo } from '@atb/components/typography';
+import { motion } from 'framer-motion';
+import { MonoIcon } from '@atb/components/icon';
+import { TransportIconWithLabel } from '@atb/components/transport-mode/transport-icon';
+import { GeocoderFeature } from '@atb/page-modules/departures';
+import { TransportModeFilterOption } from '@atb/components/transport-mode-filter/types';
+import { nextTripPatterns } from '@atb/page-modules/assistant/client';
+import { createTripQueryObject } from '@atb/page-modules/assistant/utils';
+import { useEffect, useState } from 'react';
+import { getInitialTransportModeFilter } from '@atb/components/transport-mode-filter/utils';
+import { Button } from '@atb/components/button';
+
+export type TripProps = {
+  initialFromFeature: GeocoderFeature;
+  initialToFeature: GeocoderFeature;
+  initialTransportModesFilter: TransportModeFilterOption[] | null;
+  trip: TripData;
+  departureType: 'arrival' | 'departure';
+};
+
+function tripPatternsWithTransitionDelay(tripPatterns: TripPattern[]) {
+  return tripPatterns.map((tripPattern, i) => ({
+    ...tripPattern,
+    transitionDelay: i * 0.1,
+  }));
+}
+
+export function Trip({
+  initialFromFeature,
+  initialToFeature,
+  initialTransportModesFilter,
+  trip,
+  departureType,
+}: TripProps) {
+  const { t } = useTranslation();
+  const [tripPatterns, setTripPatterns] = useState(
+    tripPatternsWithTransitionDelay(trip.tripPatterns),
+  );
+  const [nextPageCursor, setNextPageCursor] = useState(trip.nextPageCursor);
+  const [previousPageCursor, setPreviousPageCursor] = useState(
+    trip.previousPageCursor,
+  );
+  const [isFetchingTripPatterns, setIsFetchingTripPatterns] = useState(false);
+
+  useEffect(() => {
+    setTripPatterns(tripPatternsWithTransitionDelay(trip.tripPatterns));
+    setNextPageCursor(trip.nextPageCursor);
+    setPreviousPageCursor(trip.previousPageCursor);
+  }, [trip]);
+
+  const fetchNextTripPatterns = async () => {
+    setIsFetchingTripPatterns(true);
+    const tripQuery = createTripQueryObject(
+      initialFromFeature,
+      initialToFeature,
+      getInitialTransportModeFilter(initialTransportModesFilter) || undefined,
+    );
+    const cursor =
+      departureType === 'arrival' ? previousPageCursor : nextPageCursor;
+    const trip = await nextTripPatterns(tripQuery, cursor);
+    setTripPatterns([
+      ...tripPatterns,
+      ...tripPatternsWithTransitionDelay(trip.tripPatterns),
+    ]);
+    departureType === 'arrival'
+      ? setPreviousPageCursor(trip.previousPageCursor)
+      : setNextPageCursor(trip.nextPageCursor);
+
+    setIsFetchingTripPatterns(false);
+  };
+  return (
+    <>
+      {tripPatterns.map((tripPattern, i) => (
+        <TripPattern
+          key={`tripPattern-${tripPattern.expectedStartTime}-${i}`}
+          tripPattern={tripPattern}
+          delay={tripPattern.transitionDelay}
+        />
+      ))}
+      <Button
```

Comment:
The button type used in Figma doesn't exist and this should be changed after https://github.com/AtB-AS/kundevendt/issues/12072 is done

Reply:
Are we sure the design isn't the one needing to change? 🤔